.github/workflows/CICD-pr.yml

```diff
@@ -0,0 +1,8 @@
+name: Rails CI/CD Pull Request
+
+on:
+  pull_request:
+
+jobs:
+  build-and-test:
+    uses: ./.github/workflows/CI-build-test.yml
```

Comment:
Do we want to notify Slack about PR test runs?

Reply:
This workflow is rather for external devs who will fork our repo. If they create PR, it will run the tests but that's all.
I don't think we want to receive slack notifications for these PRs.
For a staff dev, we are commiting in our repo, so the branch workflow will run and we will get slack notifications

Reply:
I think it would be useful to get slack notifications for PRs, both from external devs and internally. There could be conflicts with what we developed in a branch when it comes time to merge into main and the PR might fail when the branch did not